assets/i18n/qqq.json

```diff
@@ -277,6 +280,7 @@
   "I18N_SPLASH_TITLE": "The main title shown in Oppia's landing page - This is an English idiom, it can be changed for a local common saying with similar meaning. It should not exceed one line in length.",
   "I18N_TOPNAV_ABOUT": "Text displayed in the navigation bar. - When the user hovers over the button, a list with several information pages is displayed.",
   "I18N_TOPNAV_ABOUT_OPPIA": "Text displayed inside drop-down menu - The list is shown after the user hovers over the I18N_TOPNAV_ABOUT button. When the option is clicked, the about oppia page is loaded which contains a more general description of the site.",
+  "I18N_TOPNAV_GET_STARTED": "Text displayed inside a drop-down menu. - The list is shown after the user hovers over the I18N_TOPNAV_ABOUT button. When the option is clicked, the Get Started page is loaded.",
```

Comment:
Ditto -- wrong place. Should be alphabetical.

A good way to check: `bash scripts/run_backend_tests.sh --test_target=core.controllers.base_test`